x-pack/solutions/security/plugins/security_solution/public/management/components/console/components/command_list.test.tsx

```diff
@@ -35,7 +36,9 @@ describe('When rendering the command list (help output)', () => {
       renderAndOpenHelpPanel = (props) => {
         render(props);
         helpPanelSelectors = getHelpSidePanelSelectorsAndActionsMock(renderResult);
-        consoleSelectors.openHelpPanel();
+        act(() => {
+          consoleSelectors.openHelpPanel();
```

Comment:
Thanks for the changes. 
This component actually defines several helper utilities (this one included) here:

https://github.com/elastic/kibana/blob/da25d13a2ac5be97216e202bf5d6bda6ff920b3b/x-pack/solutions/security/plugins/security_solution/public/management/components/console/mocks.tsx#L73-L107



<br/>
<br/>

It would be better to add the `act()` wrapper to all of the selectors/actions utility functions in the file above instead of in the individual test files, as that would also take care of any future tests that might use those selector utilities.